package.json

```diff
@@ -6,7 +6,8 @@
     "packages/*"
   ],
   "scripts": {
-    "start": "openmrs develop --sources 'packages/esm-*-app'",
+    "start": "openmrs develop --backend https://ugandaemr-backend.mets.or.ug --sources 'packages/esm-*-app'",
+    "start2": "openmrs develop --backend https://ohri-dev.globalhealthapp.net --sources 'packages/esm-*-app'",
```

Comment:
@pirupius  can we leave the default backend or point it to [ugandaemr](https://ugandaemr-backend.mets.or.ug/)

Reply:
Sorry about this, i added the second one cause the ugandaemr backend hadn't been updated but will remove this